config/settings/base.py

```diff
@@ -357,7 +357,7 @@
 
 CIRCLES_HUB_ADDRESS = env("HUB_ADDRESS", default="0x" + "0" * 39 + "6")
 GRAPH_NODE_ENDPOINT = env("GRAPH_NODE_ENDPOINT", default="")
-MIN_TRUST_CONNECTIONS = env("MIN_TRUST_CONNECTIONS", default=3)
```

Comment:
That's one option for changing the variable value. Another option is setting the value of `MIN_TRUST_CONNECTIONS` in the `.env` file and deploying the service with any code change but using the value en the `.env` file. I prefer the second option since it does not require a new service release. What do you think?

Reply:
Full ack.

Reply:
Cool, then you need to add the variable in the helm templates in https://github.com/CirclesUBI/circles-iac/tree/main/helm/circles-infra-suite

Reply:
check here: https://github.com/CirclesUBI/circles-iac/pull/143